src/tqec/templates/base.py

```diff
@@ -57,10 +62,18 @@ def __init__(self, shape: BaseShape) -> None:
         Each template should have a shape, represented by a Shape instance, and encoding
         how the template scales and its plaquettes.
 
+        The default increments define the distance between two plaquettes.
+        For example a default_x_increment of 2 means that two 2x2 plaquettes will share
```

Comment:
Shouldn't it be `3x3` plaquettes that share a common edge (i.e., common qubits)?

Reply:
I was wondering that, too. But from the slides, we have plaquettes touching on two qubits, so wouldn't that make each plaquette 2x2?

Reply:
There is a qubit in the middle that is used to measure the stabiliser. On the qubit grid, the basic square plaquettes are 3x3. I think saying 2x2 might be more prone to errors by the user than 3x3. Maybe let's also wait for another opinion on that?